src/explore-education-statistics-admin/src/pages/bau/components/DeleteUserModal.tsx

```diff
@@ -0,0 +1,26 @@
+import ModalConfirm from '@common/components/ModalConfirm';
+import React, { ReactNode } from 'react';
+
+interface Props {
+  triggerButton: ReactNode;
+  onConfirm: () => void;
+}
+
+const DeleteUserModal = ({ triggerButton, onConfirm }: Props) => {
+  return (
+    <ModalConfirm
```

Comment:
Hey! This is absolutely fine to do, and works perfectly well. I just wanted to add my personal opinion to give some food for thought on your frontend journey!

We didn't have a **DeleteUserButton** previously, I think for the same reasons I would not create a **DeleteUserModal**. As the **DeleteUserModal** doesn't do much other than simply return another component, I think I would avoid creating a new component and use the **ModalConfirm** in **BauUsersPage** directly. Doing so will have less lines added and less files added. However, if this modal is to appear across the service, then I'd opt for the component separation for easier tweaking later on. Hope this helps!

Reply:
Thanks Stuart!

I completely agree, that sounds like a more sensible approach to me! I think I did it like this because I saw that we split out a custom component for the same thing elsewhere, which was also used by only one other component. I was torn at the time, but your comment has now confirmed in my mind that it's better to **not** split out a custom component for this, as it is only used in one place :)